src/engine/test/source/builder/builders/opBuilderKVDBGetArray_test.cpp

```diff
@@ -0,0 +1,235 @@
+#include <any>
+#include <memory>
+#include <vector>
+
+#include <gtest/gtest.h>
+#include <json/json.hpp>
+
+#include <baseTypes.hpp>
+#include <defs/mocks/failDef.hpp>
+#include <logging/logging.hpp>
+#include <schemf/mockSchema.hpp>
+
+#include <kvdb/kvdbManager.hpp>
+#include <opBuilderKVDB.hpp>
+
+#include <metrics/metricsManager.hpp>
+
+using namespace metricsManager;
+
+namespace
+{
+using namespace base;
+using namespace builder::internals::builders;
+
+std::filesystem::path uniquePath()
```

Comment:
Could we relocate this functionality to a common place to avoid duplicating the code and be able to reuse it when needed?

Reply:
Not really, these tests are poorly located and it is necessary to move them to the next iteration of KVDB so they will not share files.